runtime/src/lib.rs

```diff
@@ -1062,6 +1064,64 @@ where
 					amount,
 				)?;
 			},
+			//mint
```

Comment:
Nit: let's make it more expressive and add the parameters to the comment
```suggestion
			// mint(currency, recipient, amount)
```